API/controllers/entityController.go

```diff
@@ -506,7 +508,130 @@ func HandleGenericObjects(w http.ResponseWriter, r *http.Request) {
 
 			return imageIDToUrl(u.EntityStrToInt(entityStr), object)
 		})
+		u.Respond(w, u.RespDataWrapper("successfully processed request", matchingObjects))
+	}
+}
+
+// swagger:operation POST /api/objects/search Objects HandleComplexFilters
+// Get all objects from any entity that match the complex filter. Return as a list.
+// Wildcards can be used on any of the parameters present in query with equality and inequality operations.
+//
```

Comment:
maybe just remove the explication of wildcard from here and mention "check endpoint X for info on wildcards"

Reply:
Done!